docs/content/patterns/alz/Policy-Initiatives.md

```diff
@@ -12,112 +12,124 @@ This document details the ALZ-Monitor Azure policy initiatives leveraged for dep
-| Deploy_ERCIR_BgpAvailability_Alert | [deploy-ercir_bgpavailability_alert.json](../../../services/Network/expressRouteCircuits/Deploy-ERCIR-BGPAvailability-Alert.json) | deployIfNotExists |
-| Deploy_ERCIR_ArpAvailability_Alert | [deploy-ercir_arpavailability_alert.json](../../../services/Network/expressRouteCircuits/Deploy-ERCIR-ARPAvailability-Alert.json) | deployIfNotExists |
-| Deploy_AFW_SNATPortUtilization_Alert | [deploy-afw_snatportutilization_alert.json](../../../services/Network/azureFirewalls/Deploy-AFW-SNATPortUtilization-Alert.json) | deployIfNotExists |
-| Deploy_AFW_FirewallHealth_Alert | [deploy-afw_firewallhealth_alert](../../../services/Network/azureFirewalls/Deploy-AFW-FirewallHealth-Alert.json)  | deployIfNotExists |
-| Deploy_PublicIp_BytesInDDoSAttack_Alert | [deploy-pip_bytesinddosattack_alert.json](../../../services/Network/publicIPAddresses/Deploy-PIP-BytesInDDOSAttack-Alert.json) | disabled |
-| Deploy_PublicIp_DDoSAttack_Alert | [deploy-pip_ddosattack_alert.json](../../../services/Network/publicIPAddresses/Deploy-PIP-DDOSAttack-Alert.json)  | deployIfNotExists |
-| Deploy_PublicIp_PacketsInDDoSAttack_Alert | [deploy-pip_packetsinddos_alert.json](../../../services/Network/publicIPAddresses/Deploy-PIP-PacketsInDDOS-Alert.json)  | disabled |
-| Deploy_PublicIp_VIPAvailability_Alert | [deploy-pip_vipavailability_alert.json](../../../services/Network/publicIPAddresses/Deploy-PIP-VIPAvailability-Alert.json)  | deployIfNotExists |
-| Deploy_VNET_DDoSAttack_Alert | [deploy-vnet_ddosattack_alert.json](../../../services/Network/virtualNetworks/Deploy-VNET-DDOSAttack-Alert.json)  | deployIfNotExists |
-| Deploy_activitylog_Firewall_Delete | [deploy-activitylog-AzureFirewall-Del.json](../../../services/Network/azureFirewalls/Deploy-ActivityLog-AzureFirewall-Del.json)  | deployIfNotExists |
-| Deploy_activitylog_RouteTable_Update | [deploy-activitylog-RouteTable-Update.json](../../../services/Network/routeTables/Deploy-ActivityLog-RouteTable-Update.json)  | deployIfNotExists |
-| Deploy_activitylog_NSG_Delete | [deploy-activitylog-NSG-Del.json](../../../services/Network/networkSecurityGroups/Deploy-ActivityLog-NSG-Del.json)  | deployIfNotExists |
-| Deploy_activitylog_VPNGateway_Delete | [deploy-activitylog-VPNGate-Del.json](../../../services/Network/vpnGateways/Deploy-ActivityLog-VPNG-Del.json)  | deployIfNotExists |
+| **Policy Name**| **Path to policy json file**| **Policy default effect** |
+|-----------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| Deploy_ERCIR_QosDropBitsInPerSecond_Alert           | [deploy-ercir_qosdropsbitsin_alert.json](../../../../services/Network/expressRouteCircuits/Deploy-ERCIR-QOSDropsBitsIn-Alert.json)                                   | deployIfNotExists         |
```

Comment:
The links are working now, so i´m wondering why the extra "../" was added

Reply:
It was the only way for me to verify the links in my local copy of the repo.

Reply:
I have a meeting with @JoeyBarnes this week to review the markdown linting but also review whether the additional ../ is required when testing locally. I can't get a local copy of hugo  locally for some reason to check whether the links work within hugo.